tools/check_release_notes.sh

```diff
@@ -99,27 +111,38 @@ validate_opts() {
         print_error_and_exit "PULL_NUMBER is a required option. It must match the GitHub pull request number."
         exit 1
     fi
+
+    if [ -z "${REPO_PATH:-}" ]; then
+        echo "REPO_PATH not specified. Using current working directory."
```

Comment:
nit: I'd move this down in this section, leaving the fatal errors first, and warnings like this last. You could also print the current directory, it might be useful for debugging. I know that this is used only outside prow though.